projects/plugins/boost/app/modules/optimizations/image-cdn/src/liar.ts

```diff
@@ -0,0 +1,6 @@
+import { dynamicSrcset } from './srcset';
+
+( function () {
+	const lazyImages = document.querySelectorAll( 'img[loading=lazy]' );
+	lazyImages.forEach( dynamicSrcset );
+} )();
```

Comment:
This might not be as important, at least not for this first iteration. When you change the orientation of the device - the srcset is still there - at that point the browser should be able to make a decision of which image to load based on the available sizes.

Reply:
Makes sense. I think this could be unrelated and some other issue. I'll create a separate issue if I face it again.